core/src/test/java/io/parsingdata/metal/ShorthandOverloadsTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2013-2020 Netherlands Forensic Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.parsingdata.metal;
+
+import static org.junit.Assert.assertTrue;
+
+import static io.parsingdata.metal.Shorthand.add;
+import static io.parsingdata.metal.Shorthand.and;
+import static io.parsingdata.metal.Shorthand.cat;
+import static io.parsingdata.metal.Shorthand.con;
+import static io.parsingdata.metal.Shorthand.div;
+import static io.parsingdata.metal.Shorthand.elvis;
+import static io.parsingdata.metal.Shorthand.eq;
+import static io.parsingdata.metal.Shorthand.len;
+import static io.parsingdata.metal.Shorthand.mod;
+import static io.parsingdata.metal.Shorthand.mul;
+import static io.parsingdata.metal.Shorthand.neg;
+import static io.parsingdata.metal.Shorthand.not;
+import static io.parsingdata.metal.Shorthand.nth;
+import static io.parsingdata.metal.Shorthand.offset;
+import static io.parsingdata.metal.Shorthand.or;
+import static io.parsingdata.metal.Shorthand.shl;
+import static io.parsingdata.metal.Shorthand.shr;
+import static io.parsingdata.metal.Shorthand.sub;
+import static io.parsingdata.metal.encoding.Encoding.DEFAULT_ENCODING;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import io.parsingdata.metal.data.ByteStream;
+import io.parsingdata.metal.data.ParseState;
+import io.parsingdata.metal.expression.value.SingleValueExpression;
+
+@RunWith(Parameterized.class)
+public class ShorthandOverloadsTest {
+
+    public static final ParseState PARSE_STATE = ParseState.createFromByteStream(new ByteStream() {
+        @Override public byte[] read(BigInteger offset, int length) { return new byte[0]; }
+        @Override public boolean isAvailable(BigInteger offset, BigInteger length) { return false; }
+    });
+
+    @Parameterized.Parameter public String description;
+    @Parameterized.Parameter(1) public SingleValueExpression toExecute;
+    @Parameterized.Parameter(2) public SingleValueExpression expectedResult;
```

Comment:
I tend to import these for improved readability.

```java
    @Parameter public String description;
    @Parameter(1) public SingleValueExpression toExecute;
    @Parameter(2) public SingleValueExpression expectedResult;

    @Parameters(name="SingleValueExpression {0}")
```

Reply:
And even though it's the default, in this case `@Parameter(0)` looks better than omitting the `(0)`.